### DISCUSSION TITLE
Need to download the file upload content via python script on each request

### DISCUSSION THREAD

**Discussion by satishkumarnk-create** [mitmproxy/mitmproxy#6120] [Q&A]
How can i write a all the file uploads for each request to the file storage using python script

**Comment by Prinzhorn** [comment]
Look into `flow.request.multipart_form`, it should contain all the data you need (assuming the files are uploaded using multipart). Everything beyond that (writing files) is generic Python code and you should find countless examples online.

https://docs.mitmproxy.org/stable/api/mitmproxy/http.html#Request.multipart_form
https://docs.mitmproxy.org/stable/api/events.html#HTTPEvents.request
